python/example-always-best-price.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+#
+# This example program uses walutomat API to watch a currency pair on Walutomat market and keep placing best offer. 
+# It uses price limit to avoid placing offers too close to Forex rate. It also automatically cancels all not best offers.
+#
+# Program runs on CHF_PLN pair, always tries to sell 0.10 CHF at price 0.0001 better than others, but not closer than 0.0040 from FX.
+#
+# Keep in mind that this is just an example program, you are using it at your own risk.
+#
+# Usage:
+# APIKEY=myapikey SECRET=myapisecret python example-always-best-price.py
+#
+
+import requests
```

Comment:
sortowanie importów

Reply:
`requests` to zewnętrzny lib, może warto jednak dodać `Pipfile`/`requirements.txt`